Adds QA features to user docs (#1784)
webrecorder/browsertrix: docs/user-guide/archived-items.md

## Archived Item Details

> The archived item details page is composed of six sections, though some tabs are only available for crawls and not uploads.

Reviewer: Will viewers see only some of these sections? Might be worth making a note of this if so

Author: QA page is hidden from viewers???  Not really sure why?  It currently has the pages list which is presumably useful for them to some extent, though all the actions on that page should be hidden from viewers.

Reviewer 2: Yes, for now, since we don't have a way of making it read-only, since running assistive QA + manual review are not read-only actions. We may need a separate role that allows QA, in the future, but for now this should be alright.

Author: Well the pages list and rankings should ideally be viewable for viewers, but fair enough, not a high priority issue (IDK if we even have a single user with the viewer role on prod haha)

Perhaps something to fix in the next iterations?  I like the idea of a QA role!  Will update with a note.

Reviewer 3: What's the reason for needing to specify the number of sections? IMO specifying the number makes the count feel more important than it is, especially when it's dependent on whether QA is enabled, what the archived item type is, and maybe other factors in the future. How about something like:
```
Archived item details are organized into the following sections.
```
And then "(crawl-only)" next to the heading for each applicable subsection that describes the tab content.

### Overview

The Overview tab displays the item's metadata and statistics associated with its creation process.

Metadata can be edited by pressing the pencil icon at the top right of the metadata section to edit the item's description, tags, and collections it is associated with.

### Quality Assurance

> The Quality Assurance tab displays a list of crawled pages, overall stats collected through crawl analysis, and a list of all analysis runs. This tab is not available for uploaded archived items.

Reviewer 3: This could lead more with how "list of crawled pages" and "overall stats" is relevant to a QA process. Maybe just by bringing up some of the keywords in the second paragraph earlier? Something like:

```diff
+ The Quality Assurance tab displays quality assurance data that is collected from manual reviews and analysis runs. This is where you can start new analysis runs, view quality metrics from older runs, and delete runs.

The pages list provides a record of all pages within the archived item, as well as any ratings or notes given to the page during review. If analysis has been run, clicking on a page in the pages list will go to that page in the review interface.
```

> The Quality Assurance tab displays a list of crawled pages, overall stats from collected through crawl analysis, and a list of all analysis runs. This tab is not available for uploaded archived items.

Reviewer: ```suggestion
The Quality Assurance tab displays a list of crawled pages, overall stats collected through crawl analysis, and a list of all analysis runs. This tab is not available for uploaded archived items.
```

> The pages list provides a record of all pages within the archived item and any ratings or notes assigned to the page during review. If analysis has been run, clicking on a page in the pages list will show that page in the review interface.

Reviewer: ```suggestion
The pages list provides a record of all pages within the archived item, as well as any ratings or notes given to the page during review. If analysis has been run, clicking on a page in the pages list will go to that page in the review interface.
```

#### Crawl Analysis

Running crawl analysis will re-visit all pages within the archived item, comparing the data collected during analysis with the data collected during crawling. Crawl analysis runs with the same workflow limit settings used during crawling.

> Crawl analysis can be run multiple times, though it will only yield different results if the crawler, or [ReplayWeb.page](https://replayweb.page/docs/) — the software responsible for playback of archived web content — has been updated between runs. Analysis run data can be downloaded or deleted from the _Analysis Runs_ tab. While they are stored as WACZ files, analysis run WACZs only contain analysis data and may not open correctly or be useful in other programs that replay archived content.

Reviewer 2: ```suggestion
Crawl analysis can be run multiple times, though results should only differ if the crawler version has been updated between runs. The analysis process is being constantly improved and future analysis runs should produce better results. Analysis run data can be downloaded or deleted from the _Analysis Runs_ tab. While they are stored as WACZ files, analysis run WACZs only contain analysis data and may not open correctly or be useful in other programs that replay archived content.
```

Author: Made some very minor changes to your suggestion for grammar.  Changed "QA workflow" to "analysis process" as we don't actually refer to this as "QA" in the app.

Do we want to completely exclude ReplayWeb.page updates as a variable here?  When replay bugs are fixed, that seems like the determining factor?  ...That said AFAIK it's built into the crawler now and so maybe this is a good change and it would be best to just list the version update for that to not complicate things?

Reviewer 2: Yes, RWP is not changed individually but via crawler versions, so want to avoid potential confusion as to what needs to be updated specifically.

Once a crawl has been analyzed — either fully, or partially — it can be reviewed by pressing the _Review Crawl_ button. For more on reviewing crawls and how to interpret analysis data, see: [Crawl Review](review.md).

`Paid Feature`{ .badge-green }

> Like running a crawl workflow, running crawl analysis also uses execution time. Crawls and crawl analysis share the same concurrent crawling limit, however crawl analysis runs will be paused in favor of new crawls if the concurrent crawling limit is reached.

Reviewer: ```suggestion
Like running a crawl workflow, running crawl analysis also uses execution time. Crawls and crawl analysis share the same concurrent crawling limit, but crawl analysis runs will be paused in favor of new crawls if the concurrent crawling limit is reached.
```
